REQUIRE

```diff
@@ -1,2 +1,2 @@
 julia 0.6
-Compat 0.9.5
+Compat 0.28.0
```

Comment:
I put 0.28 since 0.27 has a weird warning that was fixed in https://github.com/JuliaLang/Compat.jl/pull/383